docs/copilot/getting-started.md

```diff
@@ -1,56 +1,31 @@
 ---
-Order: 10
+Order: 3
 Area: copilot
-TOCTitle: Getting started tutorial
+TOCTitle: Getting Started Tutorial
 ContentId: 37fd3bd2-4209-49f6-bec5-c544d6b1b289
 PageTitle: GitHub Copilot getting started
 DateApproved: 02/28/2024
 MetaDescription: Get started with GitHub Copilot in Visual Studio Code and create your first AI-powered suggestions in the editor.
 ---
 # Getting started with GitHub Copilot in VS Code
 
-This tutorial walks you through setting up the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) extension, and discovering the key features of GitHub Copilot in Visual Studio Code. Learn how to get started with AI-powered code suggestions, use chat conversations to refactor your code, and fix code errors with smart actions.
+This tutorial walks you through using the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) extension to discover the key features of GitHub Copilot in Visual Studio Code. Learn how to get started with AI-powered code suggestions, use chat conversations to refactor your code, and fix code errors with smart actions.
 
-In this tutorial, you're using GitHub Copilot to help you create a `Calculator` class in TypeScript. 
+In this tutorial, you're using GitHub Copilot to help you create a `Calculator` class in TypeScript.
 
 > **Note:** While we're using TypeScript for this tutorial, please note that Copilot is also trained on numerous other languages and a wide variety of frameworks.
 
 For an overview of what you can do with GitHub Copilot in VS Code, see the [GitHub Copilot Overview](/docs/copilot/overview.md).
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/2q0BoioYSxQ" title="GitHub Copilot Best Practices (what not to do)" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-## Set up VS Code for GitHub Copilot
+## Prerequisites
 
-### Step 1: Set up your GitHub Copilot subscription
+* To use GitHub Copilot in VS Code, you must have the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) extension. When you install this extension, the [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) extension is also installed.
 
-If you want to use GitHub Copilot, you either need an active subscription for GitHub Copilot in your personal account, or you need to be assigned a seat in a subscription managed by an organization or enterprise.
+* To use GitHub Copilot, you must have an active subscription for GitHub Copilot in your personal account, or you need to be assigned a seat by your organization.
```

Comment:
Feels like there should be a link to account setup at this point?

Reply:
I realize now that linking to `setup.md` walks users through this. I'd maybe more specific so that I know I don't need to go exploring through the docs to find subscription info. I'll leave a suggestion on the next line.